server.js

```diff
@@ -39,14 +40,28 @@ client[custom.clock_tolerance] = 300; // to allow a 5 minute clock skew for veri
 // This method is NOT recommended for use in production systems.
 let accessToken;
 
+const claims = {
```

Comment:
**suggestion:** We may want to include a comment which explains that these are just _some_ of the possible `claims` which can be requested.

This suggestion is analogous to what we do for the `scope` parameter which has this comment to help folks understand what they are reading:

```javascript
// These are the OpenID Connect scopes that you'll need.
```